content/docs/intro/vs/cloud_templates.md

```diff
@@ -11,18 +10,26 @@ menu:
 aliases: ["/docs/reference/vs/cloud_templates/"]
 ---
 
+What is Pulumi?
+
+Pulumi offers a multi-cloud alternative to typical, more restrictive infrastructure-as-code template solutions, such as AWS CloudFormation. Pulumi allows you to streamline processes through managing resources from different platforms all in one place, using real code, in your favorite language.
+
+What is CloudFormation?
+
 All major cloud providers offer their own form of infrastructure-as-"code" solution, typically by way of JSON or
 YAML-based templating solutions. This includes AWS CloudFormation and Azure Resource Manager (ARM) templates.
 
 These markup-based configuration files are often uploaded to a hosted service in the target cloud, where a hosted
 service will then process the files to create, update, or delete resources as necessary.
 
+The Pulumi Difference
+
 Pulumi's model shares a lot with these systems. Although Pulumi programs are written in imperative, familiar languages,
 they are ultimately evaluated to produce a similar set of create, update, or delete operations for your cloud resources.
 
-This is where Pulumi begins to diverge from these other solutions, however.
+This is where Pulumi begins to diverge from these other solutions such as AWS CloudFormation, however.
 
-Pulumi lets you use your favorite languages, instead of bespoke templating solutions. These typically use syntaxes that
+Pulumi lets you use your favorite languages — complete control rather than bespoke templating solutions such as CloudFormation. These templates typically use syntaxes that
```

Comment:
This sentence is no longer clear. Should the dash be a colon? And "complete control" and "bespoke templating solutions" are not directly comparable enough to be separated with a "rather than". (In other words, languages and templating solutions are comparable, as they're both technologies. Control is not a technology.)

Reply:
It's probably fine to leave as is then, as long as we can still get the "CloudFormation" part in there.